packages/backend/src/index.ts

```diff
@@ -37,6 +37,7 @@ import scaffolder from './plugins/scaffolder';
 import search from './plugins/search';
 import sonarqube from './plugins/sonarqube';
 import techdocs from './plugins/techdocs';
+import permission from './plugins/permission';
```

Comment:
Should the filename and import be updated to `permissions` since it will have more than one permission?

Reply:
Right now, this adds a single default permission policy to the showcase app. So I think `permission` should suffice. Whenever the RBAC plugin is introduced, that will handle the creation on several permission policies and I believe that it might warrent a change then.

Reply:
Looks like renaming it to `permissions` might not be possible due to [line 55](https://github.com/janus-idp/backstage-showcase/blob/11ca8dd941c149311f4815380b10ce0d11c075e2/packages/backend/src/index.ts#L55) and [line 76](https://github.com/janus-idp/backstage-showcase/blob/11ca8dd941c149311f4815380b10ce0d11c075e2/packages/backend/src/index.ts#L76). We could do something like `permissionPolicy` instead